src/sqlfluff/dialects/dialect_hive.py

```diff
@@ -265,6 +265,22 @@ class ArrayTypeSegment(ansi.ArrayTypeSegment):
     )
 
 
+class EqualsSegment(ansi.EqualsSegment):
+    """Equals operator.
+
+    Hive allows double equals:
+    https://cwiki.apache.org/confluence/display/Hive/Hive+Operators
+    """
+
+    match_grammar: Matchable = OneOf(
+        Ref("RawEqualsSegment"),
+        Sequence(
```

Comment:
This can be simplified to a sequence with
```
Ref("RawEqualsSegment"),
Ref("RawEqualsSegment", optional=True),
```

Reply:
Done